<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageBoxEx</class>
 <widget class="QDialog" name="MessageBoxEx">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="TitleBar" name="titleBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="button">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="infoTE">
   <property name="geometry">
    <rect>
     <x>69</x>
     <y>40</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="iconL">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="button2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleBar</class>
   <extends>QGroupBox</extends>
   <header location="global">ui/titlebar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
